frontend/src/topic/HomeScreen/views/index.js

```diff
@@ -3,23 +3,36 @@ import Helmet from 'react-helmet';
 import PropTypes from 'prop-types';
 import React from 'react';
 import { connect } from 'react-redux';
+import { Redirect } from 'react-router-dom';
 
 import { pageTitles } from '../../../common/utils/labels';
 import {
     setNavbarStateAction,
 } from '../../../common/action-creators/navbar';
+import {
+    activeProjectSelector,
+    currentUserProjectsSelector,
+} from '../../../common/selectors/domainData';
 
 import styles from './styles.scss';
 
 const propTypes = {
     setNavbarState: PropTypes.func.isRequired,
+    activeProject: PropTypes.object.isRequired, // eslint-disable-line
```

Comment:
activeProject is getting numeric values

![screenshot from 2017-10-24 12-08-55](https://user-images.githubusercontent.com/4931541/31927558-6299369a-b8b4-11e7-9427-3536a806a395.png)